load/InputFileset.py

```diff
@@ -116,6 +126,32 @@ def toString(self):
 			results.append(file.toString() + "\n")
 		return " ".join(results)
 
+	def getSetTypeCode(self):
+		if self.typeCode == "video":
+			return "video_stream"
+		elif self.typeCode == "audio":
+			code = self.filesetId[7:9]
+			if code == "1D":
+				return "audio"
+			elif code == "2D":
+				return "audio_drama"
+			else:
+				code = self.filesetId[8:10]
+				if code == "1D":
+					return "audio"
+				elif code == "2D":
+					return "audio_drama"
+				elif self.filesetId == "N1TUVDPI":
```

Comment:
I suspect these very specific cases should be removed, right? I don't see any reason to keep them in

Reply:
@bradflood I have removed these specific cases.